### ISSUE TITLE
Unexpected background with gvim (gruvbox/windows)

### ISSUE THREAD

**Issue by wsdjeg** [blueyed/vim-diminactive#40]
![qq 20171129142740](https://user-images.githubusercontent.com/13142418/33361066-7c0db848-d511-11e7-9858-7e2bb43b8c7d.png)

 I get red bg in the inactive windows, I think it is issue with this plugin, does it works well in your gvim?

gvim8 + window7

**Comment by blueyed** [comment]
Yes - more or less (my config is not really optimized for gvim).
Does it work with non-gui Vim and/or Neovim for you?

**Comment by wsdjeg** [comment]
It works well for me in terminal vim/neovim, and  also works well in gvim8 in archlinux,

**Comment by blueyed** [comment]
Maybe an issue with gruvbox on Vim/win then in general?!

**Comment by wsdjeg** [comment]
ok, I will test some other colorscheme if I have windows pc.
